.gitignore

```diff
@@ -1,3 +1,3 @@
 _build/*
 deps/*
-.env*
+.env
```

Comment:
Buuuuut we don't want to commit .env.development or .env.production

I think we probably want to ignore `.env*` and make an exception for `.env.sample`